README.md

```diff
@@ -21,10 +21,10 @@
 
 ## Prerequisites
 - Ruby (3.1.0 or later)
-- Postgres
+- Postgres (note: 14 is the version)
   ```
-  brew install postgres
-  brew services start postgresql
+  brew install postgresql@14
```

Comment:
@lizgarciao thank you. Can we keep the version agnostic? For example instead of a specific version add version-number. 
Thanks for taking the time to update the readme!